presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigDeployer.java

```diff
@@ -115,11 +115,21 @@ public void deployDefaultConfig(Path configDir, Path dataDir, Path pluginDir)
                 // TODO: Include node.id in install requests
                 updateProperty(configDir.resolve("node.properties"),
                         "node.id", randomUUID().toString());
+                /**
+                 * To prevent conflicts between Presto-Manager configs and
+                 * Presto configs present in {@link #defaultConfig} directory
+                 */
+                Path nodePropertiesPath = configDir.toAbsolutePath().resolve("node.properties");
```

Comment:
Let's say user provides `configDir` location as `/etc/presto` in the Presto-Manager config files.
But in the default configs for Presto the `configDir` is `/usr/presto`. This would cause a conflict.
I'm updating these variables to prevent these conflicts.